### PR TITLE
kubeadm: update the setup documentation for 1.15

### PR DESCRIPTION
**Pull request by neolit123**
placeholder for 1.15

should hold changes under /setup/independent/...

xref https://github.com/kubernetes/kubeadm/issues/1577

@kubernetes/sig-cluster-lifecycle-pr-reviews 
/cc @fabriziopandini 
/sig cluster-lifecycle
/milestone v1.15

### CODE REVIEW

**Comment by neolit123** [comment]
potentially ready for review.

File: content/en/docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -33,17 +33,17 @@ installing deb or rpm packages. The responsible SIG for kubeadm,
...
 | Implementation            | GA             |
-| Config file API           | beta           |
+| Config file API           | Beta           |
 | CoreDNS                   | GA             |
-| kubeadm alpha subcommands | alpha          |
-| High availability         | alpha          |
-| DynamicKubeletConfig      | alpha          |
+| kubeadm alpha subcommands | Alpha          |
+| High availability         | Beta           |
+| DynamicKubeletConfig      | Alpha          |
```
Comment (fabriziopandini) [inline content/en/docs/setup/independent/create-cluster-kubeadm.md]: Not sure, but if I got this right from Tim, we are not considering phases GA...

Comment (fabriziopandini) [reply content/en/docs/setup/independent/create-cluster-kubeadm.md]: Automatic Certificate Copy: Beta

Comment (neolit123) [reply content/en/docs/setup/independent/create-cluster-kubeadm.md]: > Not sure, but if I got this right from Tim, we are not considering phases GA...

if not GA do we have a State for them?

> Automatic Certificate Copy: Beta

isn't it a core functionality of HA going to Beta?

Comment (fabriziopandini) [reply content/en/docs/setup/independent/create-cluster-kubeadm.md]: @neolit123 
Ok, agreed with your comments (no changes required)

**Pull request merged**